core/dbt/events/types.py

```diff
@@ -1414,11 +1414,11 @@ def message(self) -> str:
 
 @dataclass
 class DepsStartPackageInstall(InfoLevel, Cli, File):
-    package: str
+    package_name: str
     code: str = "M014"
 
     def message(self) -> str:
-        return f"Installing {self.package}"
+        return f"Installing {self.package_name}"
```

Comment:
An alternative is to take the package value and in the message method, call `{self.package.name}`. It's a little more work but its an improvement from passing strings which I think we should avoid when possible.